project/eScavenge/test_Game.py

```diff
@@ -573,7 +575,26 @@ def test_get_status_negative_time(self):
                          'get_status did not print the proper stats!')
         self.assertEqual(stat_str.format(2, 2, 0, 1, 3, "0:00:00", "0:00:00"), self.game.get_status(now, 'ghi'),
                          'get_status did not print the proper stats!')
-
+    def test_get_status_place(self):
+        TeamFactory.get_team('xyz', '123', self.game)
+        temp = self.game.teams.get(username='abc')
+        temp.points = 20
+        temp.clue_time = timezone.now()
+        temp.save()
+        temp = self.game.teams.get(username='ghi')
+        temp.points = 30
+        temp.save()
+        temp = self.game.teams.get(username='xyz')
+        temp.points = 10
+        temp.save()
+        now = temp.clue_time - datetime.timedelta(hours=1)
```

Comment:
This test feels a little weird. Especially with this clue_time setting, since the last temp did not get his clue_time changed now is the game start time minus an hour, and team abc has a very slightly different clue_time. What exactly are those changes for? Also, you're setting 3 teams stats, but only checking one. If all we want to check is one then you only need to change one team (set them in the lead and the other two teams will be in second.

Reply:
None of this is technically wrong, it just seems like a lot of changing for no reason.